ginga/ImageView.py

```diff
@@ -3155,15 +3155,18 @@ def render_image(self, rgbobj, dst_x, dst_y):
 
     def reschedule_redraw(self, time_sec):
         """Reschedule redraw event.
-        This must be implemented by subclasses.
+
+        This should be implemented by subclasses.
 
         Parameters
         ----------
         time_sec : float
             Time, in seconds, to wait.
 
         """
-        self.logger.warning("Subclass should override this abstract method!")
+        # subclass implements this method to call delayed_redraw() after
+        # time_sec
+        self.delayed_redraw()
```

Comment:
Do you really mean this, from the comment?
```suggestion
        self.delayed_redraw(time_sec)
```

Reply:
No, `time_sec` is set elsewhere.  I will update the comment.

Reply:
Changing the comment to:
```python
# subclass implements this method to call delayed_redraw() after
# time_sec.  If subclass does not override, redraw is immediate.
```